camera_frame    : "camera_link"
project_path    : find(ros_visual)
results_topic   : "/fusion/results"
events_topic   : "/decision_making/events"
csv_fields      : "Timestamp, Rect_Id, Annotation"
min_depth       : 0
max_depth       : 6000
history_size    : 7
create_directory: false
write_csv       : false
display         : false
